sspl_test/run_test.py

```diff
@@ -43,7 +43,7 @@
 storage_type = None
 
 try:
-    setup_info = subprocess.Popen("sudo /usr/bin/provisioner get_setup_info", shell=True,
+    setup_info = subprocess.Popen(['sudo', 'provisioner', 'get_setup_info'], shell=False,
```

Comment:
Why changed shell=False?

Reply:
Its all about how to execute the command.

For shell=True, it takes single command like subprocess.Popen("sudo /usr/bin/provisioner get_setup_info", shell=True)

For shell=False, it takes single command separated and in a list like subprocess.Popen(['sudo', 'provisioner', 'get_setup_info'], shell=False)

Reply:
shell=True is not a god practice to use according to python documentation. If we do this, it will create again its own shell process.

Reply:
ok